docs/openapi.yaml

```diff
@@ -698,203 +884,258 @@ components:
+        format: uuid
+    instanceTypeParam:
+      name: instance_type
+      in: query
+      description: "Returns document with this instanceType."
+      example: JournalArticle
+      required: false
+      schema:
+        $ref: "#/components/schemas/CategoryEnum"
+    instanceTypeNotParam:
+      name: instance_type_not
+      in: query
+      description: "Returns document without this instanceType."
+      example: JournalArticle
+      required: false
+      schema:
+        $ref: "#/components/schemas/CategoryEnum"
+    instanceTypeShouldParam:
+      name: instance_type_should
+      in: query
+      description: "Returns document with these instanceTypes."
+      example: JournalArticle
+      required: false
+      schema:
+        $ref: "#/components/schemas/CategoryEnum"
     institutionParam:
       name: institution
       in: query
-      description: "returns document with Id, name or acronym of the institution the contributors belong to"
-      example: [ 185, UiO, University of Oslo ]
+      description: "Returns document with this institution id (URI)."
+      examples:
+        Id:
+          value: "185"
+        Acronyms:
+          value: "UiO"
+        Name:
+          value: "University of Oslo"
       required: false
       schema:
         type: string
     institutionNotParam:
       name: institution_not
       in: query
-      description: "returns document without Ids, names or acronyms of the institution the contributors belong to, seperated by comma (+)."
-      example: [ 185, UiO, University of Oslo ]
+      description: "Returns document without Ids, names or acronyms of the institution the contributors belong to."
+      examples:
+        Id:
+          value: "185"
+        Acronyms:
+          value: "UiO"
+        Name:
+          value: "University of Oslo"
       required: false
       schema:
         type: string
     institutionShouldParam:
       name: institution_should
       in: query
-      description: "returns document with Ids, names or acronyms of the institution the contributors belong to, seperated by comma (+)."
-      example: [ 185, UiO, University of Oslo ]
+      description: "Returns document with Ids, names or acronyms of the institution the contributors belong to."
+      examples:
+        Id:
+          value: "185"
+        Acronyms:
+          value: "UiO"
+        Name:
+          value: "University of Oslo"
       required: false
       schema:
         type: string
     isbnParam:
       name: isbn
       in: query
-      description: "returns document with this ISBN id"
+      description: "Returns document with this ISBN id"
+      example: "978-3-16-148410-0"
       required: false
       schema:
         type: string
     isbnNotParam:
       name: isbn_not
       in: query
-      description: "returns document without these ISBN ids, seperated by comma (+)."
+      description: "Returns document without these ISBN ids."
+      example: "978-3-16-148410-0"
```

Comment:
Give an example of isbn without dashes? I think we support that?

Reply:
Actually, the search api needs to be able to wash incomming search phrases, for formatting, and in some cases add formatting